contrib/kube-prometheus/manifests/grafana-deployment.yaml

```diff
@@ -16,7 +16,7 @@ spec:
         app: grafana
     spec:
       containers:
-      - image: grafana/grafana:5.2.1
+      - image: grafana/grafana:5.2.4
```

Comment:
The default Grafana image actually needs to be bumped here: https://github.com/brancz/kubernetes-grafana/blob/9a20f81c9007e4c7409dd0b3edda1a7a78ad2c63/grafana/grafana.libsonnet#L8

As this manifest is generated from jsonnet and more specifically the brancz/kubernetes-grafana repository is what defines the Grafana dependency.